src/app/features/public-form/public-form.module.ts

```diff
@@ -47,6 +47,13 @@ const dynamicComponents: [string, AsyncComponent][] = [
         "app/features/public-form/edit-prefilled-values/edit-prefilled-values.component"
       ).then((c) => c.EditPrefilledValuesComponent),
   ],
+  [
+    "EditPublicformRouteComponent",
```

Comment:
We have registered these without the "Component" suffix --> "EditPublicFormRoute"

I would also tend to capitalize it "PublicForm" (although I think I understand your idea to group the naming)